Automattic/wp-calypso
# fix: Adjust mobile app CTA vertical rhythm

#### Proposed Changes

Follow up to https://github.com/Automattic/wp-calypso/pull/68062#issuecomment-1263891215. Adjust the vertical rhythm to more closely match the original mobile CTA design. 

<details><summary>Video comparison: browser vs design</summary>

https://user-images.githubusercontent.com/438664/193359958-58d1db05-07a5-49fb-89d9-b328074cdda9.mov

</details>

<details><summary>Screenshot</summary>

<img width="415" alt="mobile-cta" src="https://user-images.githubusercontent.com/438664/193360035-db337e59-c364-435b-b524-00f116e27a21.png">

</details>

#### Testing Instructions

Visit the Home path, verify the mobile CTA rendering matches the original design.

#### Pre-merge Checklist
